demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -1,8 +1,10 @@
+# isort:skip_file
```

Comment:
why do you skip isort?

Reply:
Cause, I fix what it wanted and still failed on the same file